app/src/main/java/com/example/felipe/criminalintent/CrimeActivity.kt

```diff
@@ -1,14 +1,20 @@
 package com.example.felipe.criminalintent
 
-
 import android.support.v7.app.AppCompatActivity
 import android.os.Bundle
+import android.support.v4.app.Fragment
 
 class CrimeActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_crime)
+        initFragments()
+    }
+
+    private fun initFragments() {
         var fragment = CrimeFragment()
-        supportFragmentManager.beginTransaction().add(R.id.fragmentContainer, fragment).commit()
+        fragment.doTransaction()
     }
+
+    private fun Fragment.doTransaction() = supportFragmentManager.beginTransaction().add(R.id.fragmentContainer,this).commit()
```

Comment:
the name should say what the method do, so you should call it something like, `addFragmentToFragmentContainer` or something like that

Reply:
and use multiple lines, from my point of view it's more clear